archlinux/PKGBUILD

```diff
@@ -75,6 +79,18 @@ install -D $srcdir/PKGBUILD-z-qubes-session.sh $pkgdir/etc/X11/xinit/xinitrc.d/z
 
 }
 
+package_qubes-vm-pulseaudio() {
+
+pkgdesc="Pulseaudio support for Qubes VM"
+depends=( 'alsa-lib' 'alsa-utils' 'pulseaudio-alsa'
+		'pulseaudio>=11.1' 'pulseaudio<11.2'
```

Comment:
I wonder if this could be calculated automatically at package build time (see spec file for rpm). But it isn't blocking this PR.

Reply:
It can be but I never found how to compute next minor versions using bash without leaving a bug for rare (and consequently occuring) cases.